src/CBot/CBotToken.cpp

```diff
@@ -466,8 +466,7 @@ std::unique_ptr<CBotToken> CBotToken::CompileTokens(const std::string& program)
 {
     CBotToken       *nxt, *prv, *tokenbase;
     const char*     p = program.c_str();
-    int             pos = 0;
-
+    std::size_t     pos = 0;
```

Comment:
This is not really a remark but a general observation. I get that you only changed the type here but I think people should start cleaning up around the places where they change something. We'll never get out of the mess that is the current Colobot code if we just keep on adding more mess or ignore it.